main.js

```diff
@@ -50,13 +70,34 @@ const updateBall = () => {
   }
 };
 
+function checkInputs() {
```

Comment:
This doesn't check inputs. Its registering an event handler that will in turn handle event inputs.

It should be called something like `registerInputHandler`